### DISCUSSION TITLE
Incorrect grouping of releases in feed

### DISCUSSION THREAD

**Discussion by chancez** [community/community#66530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In the screenshot below, you'll see the card for "pypa/setuptools" had 2 releases, but it incorrectly puts jqlang/jq 1.7 in the same card.

![Screenshot 2023-09-08 at 12 33 02 PM](https://github.com/community/community/assets/1400401/baf4fc60-2314-44df-8b73-00ca6a549641)

**Comment by dtinth** [comment]
encounter the same issue

<img width="724" alt="image" src="https://github.com/community/community/assets/193136/fa6db4f5-23c8-4335-a5d0-75dca1ffc6da">

- top half is https://github.com/jgraph/drawio/releases/tag/v21.7.4
- bottom half is https://github.com/ruby/ruby.wasm/releases/tag/2023-09-08-a
top half is
